### PR TITLE
resolve 3.4.1 by eliminating access to seeing errors without authentication

### PR DESCRIPTION
**Pull request by slubwama**
Paired with PR https://github.com/openmrs/openmrs-core/pull/4742 in core

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you attach screenshots of how this looks like with and without authentication?

**Comment by slubwama** [comment]
> Can you attach screenshots of how this looks like with and without authentication?

Screenshot 2024-09-09 at 03 53 25

File: omod/src/main/webapp/uncaughtException.jsp [hunk 1]
```diff
@@ -51,195 +55,207 @@
...
 
 </div>
-	
+
+
+<%
+    } else {
+
+%>
+<h2><openmrs:message code="no.authenticated.user.found"/></h2>
```
Comment (dkayiwa) [inline omod/src/main/webapp/uncaughtException.jsp:254]: This error message looks like the action was about looking for an authenticated user. What do you think of using something like `Authentication Required`?

Comment (slubwama) [reply omod/src/main/webapp/uncaughtException.jsp]: @dkayiwa resolved

File: api/src/main/resources/messages.properties [hunk 2]
```diff
@@ -6,6 +6,7 @@ legacyui.manageuser.providerAccount=Provider Account
 legacyui.manageuser.createProviderAccount=Create a Provider account for this user
 legacyui.manageuser.providerIdentfier=Provider Identifier(s):
 legacyui.manageuser.noProviderIdentifier=No Identifier Specified
+legacyui.error.aunthenticationRequired=Authentication Required
```
Comment (ibacher) [inline api/src/main/resources/messages.properties:9]: We should probably fix the spelling here `authenticationRequired`

**Comment by dkayiwa** [comment]
@slubwama i see something like `(The full error stack trace output is in the source of this page.)`
But i do not see it when i check the page source.

**Pull request merged**